developer-docs-site/docs/intro.md

```diff
@@ -6,20 +6,116 @@ sidebar_position: 0
 
-## Want to dive in?
-
-- Start with the basics, such as [Transactions and States](basics/basics-txns-states) and how [Accounts](/basics/basics-accounts) work. 
-
-- Want something a bit more interactive? You can dive right into our [tutorials](/guides/getting-started).
-
-Or you can go straight to the [aptos-core source](https://github.com/aptos-labs/aptos-core) and help us build the Layer 1 for everyone.
+:::tip Aptos Incentivized Testnet-2 (AIT-2)
+Click [here for the Aptos Incentivized Testnet-2 program](tutorials/validator-node/ait-landing). 
+:::
 
-:::info Devnet Resets
-We will reset the devnet every Thursday Morning (PST). See the latest updates in [Aptos Discord][discord].
+## Start here
+
+<div class="docs-card-container">
+<div class="row row-cols-1 row-cols-md-2a g-4">
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column">
+    <a href="#" class="card-title card-link stretched-link"> <h2>New to Aptos?</h2></a>
+    <p class="card-text">Start with this big picture, Aptos mental model, document.</p>
+</div>
+</div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column" >
+    <a href="guides/dev-environment" class="card-title card-link stretched-link"> <h2>Development setup</h2></a>
+    <p class="card-text">Begin by setting up your Aptos development environment.</p>
+</div>
+  </div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column">
+    <a href="tutorials/quickstart-tutorials/" class="card-title card-link stretched-link"> <h2>Run quickstart tutorials</h2></a>
+    <p class="card-text">Submit your first transaction, write your first Move module, deploy your first coin, and more.</p>
+</div>
+</div>
+</div>
+</div>
+
+## Concepts and guides
+
+<div class="docs-card-container">
+<div class="row row-cols-1 row-cols-md-2a g-4">
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column">
+    <a href="#" class="card-title card-link stretched-link"> <h2>Concepts</h2></a>
+    <p class="card-text">Lean the key concepts of the Aptos Blockchain, including transactions, accounts, gas and transaction fees, nodes, and more. </p>
+</div>
+</div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column" >
+    <a href="#" class="card-title card-link stretched-link"> <h2>How it works</h2></a>
+    <p class="card-text">How does interacting with the Aptos Blockchain work, how Move works on Aptos, how to create a signed transaction, and more.</p>
+</div>
+  </div>
+  <div class="col">
+    <div class="card card-body h-100 d-flex flex-column">
+    <a href="#" class="card-title card-link stretched-link"> <h2>Move on Aptos</h2></a>
+    <p class="card-text">Learn the differences between the Aptos devnet, a local testnet and the Aptos Inceitivized Testnet.</p>
+</div>
+</div>
+</div>
+</div>
+
+## Nodes
+
+<div class="docs-card-container">
```

Comment:
wow, no way to do this with markdown natively?

Reply:
Yeah, nothing out there was to my satisfaction. Also the Markdown natively doesn't support cards or any such non-textual feature. So I got down to hacking this. It works and to some extent I can control it, so better than nothing.